js/script.js

```diff
@@ -0,0 +1,102 @@
+class Exchange {
+    constructor(exchangeObject) {
+        
+    this.name   = exchangeObject.name;
+    this.symbol = exchangeObject.symbol;
+    
+    }
+ createExchangeList() {
+        const container = document.getElementById('list-container')
+        const listWrapper = document.createElement('div')
+        listWrapper.classList.add('list-group-item')
+        const exchangeList = document.createElement('a')
+        const url = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/profile/${this.symbol}`
+        fetch(url).then(response => {
+            return response.json();
+        }).then(result => {
+            const companyLogo = result[0].image
+            const companyLogoUI = document.createElement('img')
+            companyLogoUI.setAttribute('style', 'width:30px')
```

Comment:
Instead of having multipile calls on companyLogoUI (same for stockChangeUI) you can use innerHTML like so:

listWrapper.innerHTML += `<img style='width: 30px' src=${companyLogo}/>`